scripts/rollup/bundles.js

```diff
@@ -313,7 +313,7 @@ const bundles = [
   {
     bundleTypes: [FB_WWW_DEV, FB_WWW_PROD],
     moduleType: RENDERER,
-    entry: 'react-flight-dom-relay',
+    entry: 'react-transport-dom-relay',
     global: 'ReactFlightDOMRelayClient',
```

Comment:
I kept this name so we don't break the sync but we should follow up to rename this internally.